### PR TITLE
{releasing.md} Add instruction to update the title

### PR DESCRIPTION
**Pull request by romoore**
Once the release version is determined, the title of the PR should be updated and reviews requested.

### CODE REVIEW

File: contributing/releasing.md [hunk 1]
```diff
@@ -214,6 +214,11 @@ Commit the results to your branch:
 
     git commit -am "Bumped version number to $(scripts/print_version)."
     git push origin release-candidate
+    
+Update the PR title to the release version. The format is typically "vX.Y.Z" (*e.g.*, v72.0.1). 
+Once this is done, send the PR out for review. Add "material-components/core-ios-team" to the 
+list of Reviewers. Also add anyone else you think might need to review a particular set of 
+changes either in the release candidate or that you've made to fix internal test failures.
```
Comment (galiak11) [inline contributing/releasing.md]: This sentence is hard to follow. Especially the part after the "or".

Comment (romoore) [reply contributing/releasing.md]: Done

**Review by jverkoey — APPROVED** [review]
